bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "gpii-binder",
+  "version": "0.0.0",
+  "homepage": "https://github.com/GPII/gpii-binder",
+  "authors": [
+    "Tony <tony@raisingthefloor.org>"
+  ],
+  "description": "A Fluid component library that creates live bindings between model data and views.",
+  "moduleType": [
+    "amd"
```

Comment:
I don't think I'm a standard node module either, since this is describing the client-side code.  Here are the choices:

https://github.com/bower/spec/blob/master/json.md#moduletype

To me, "globals" seems most accurate.